ktor-client/ktor-client-android/jvm/src/io/ktor/client/engine/android/AndroidClientEngine.kt

```diff
@@ -88,7 +93,7 @@ public class AndroidClientEngine(override val config: AndroidEngineConfig) : Htt
 
             val content: ByteReadChannel = connection.content(callContext, data)
             val headerFields: Map<String, List<String>> = connection.headerFields
-                .mapKeys { it.key?.toLowerCase() ?: "" }
+                .mapKeys { it.key?.lowercase(Locale.getDefault()) ?: "" }
```

Comment:
Is there overload that does not need locale?

Reply:
no, it was deprecated